package.json

```diff
@@ -35,5 +40,8 @@
   },
   "bugs": {
     "url": "https://github.com/Gerrit0/typedoc-plugin-mdn-links/issues"
+  },
+  "dependencies": {
+    "axios": "^1.6.0"
```

Comment:
Am I missing something? It looks like this ought to be a dev dependency

Reply:
Yes, this was an oversight on my part -- will fix shortly